.gitignore

```diff
@@ -2,7 +2,7 @@ node_modules/
 
 /build/
 /storybook-static/
-/packages/dist-*/
+/packages/**/dist/
```

Comment:
`/packages/*/dist/` does not work as in `.eslintignore` and `.prettierignore` ?

Reply:
The question is more does `/*/` work in `.eslintignore` and `.prettierignore`... 🤣 🎣